docs/howto/how-to-use-nnfw-python-api.md

```diff
@@ -22,46 +22,19 @@ else:
     session = nnfw_session(nnpackage_path, backends)
```

Comment:
It would be good to add NOTE or Caution, that Python API is experimental yet. It can be changed.

Reply:
I'll update ASAP.

Reply:
I updated the docs for caution.

Reply:
I added my comment here github does not allow to add comment only the modified lines. `Caution` should be at top of the document, not as comment which is not noticible.

Reply:
I just updated `CAUTION` under the title.